app/controllers/admin/routes.py

```diff
@@ -174,3 +193,33 @@ def addParticipants():
 
     return render_template('addParticipants.html',
                             title="Add Participants")
+
+@admin_bp.route('/<username>/courseProposals', methods=['GET'])
+def createTable(username, unauthorized=False):
```

Comment:
Where is unauthorized coming from here? Are you using some sort of decorator somewhere?

Reply:
refactored code so unauthorized is unneeded.